src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -811,28 +811,25 @@ internal Task<T> WaitForExternalEvent<T>(string name, string reason)
             {
                 // We use a stack to make it easier for users to abandon external events
```

Comment:
This comment needs to be updated now that we're not using a stack type directly.

Reply:
This is still a stack but with a custom implementation. I've amended the comment.